R/metrics-quantile.R

```diff
@@ -253,11 +255,16 @@ interval_coverage <- function(observed, predicted,
     100 - (100 - interval_range) / 2
   ) / 100
   if (!all(necessary_quantiles %in% quantile_level)) {
-    warning(
-      "To compute the interval coverage for an interval range of ",
-      interval_range, "%, the `", toString(necessary_quantiles),
-      "` quantiles are required. Returning `NA`."
+    #nolint start: keyword_quote_linter object_usage_linter
+    cli_warn(
+      c(
+        "!" = "To compute the interval coverage for an interval range of
+        {interval_range}, the {.val {necessary_quantiles}} quantiles
```

Comment:
```suggestion
        {interval_range}%, the {.val {necessary_quantiles}} quantiles
```

the old version had a %-sign there. is it possible to get that back in like that?